SuperBuild/External_PlusLib.cmake

```diff
@@ -72,9 +72,17 @@ ENDIF()
 
 IF(PLUS_USE_OPTITRACK)
   LIST(APPEND PLUSBUILD_ADDITIONAL_SDK_ARGS
-    -DOPTITRACK_INCLUDE_DIR:PATH=${OPTITRACKSDK_INCLUDE_DIR}
-    -DOPTITRACK_LIBRARY_DIR:PATH=${OPTITRACKSDK_LIBRARY_DIR}
-    -DOPTITRACK_BINARY_DIR:PATH=${OPTITRACKSDK_BINARY_DIR}
+    -DOPTITRACK_MOTIVE_INCLUDE_DIR:PATH=${MOTIVE_API_INCLUDE_DIR}

```

Comment:
Could you make this a bit more intelligent to only send the variables that are active? This way it doesn't pollute the PlusLib CMake variable space